specification/gedcom-3-structures-1-organization.md

```diff
@@ -1188,6 +1188,7 @@ This specification does not support places where a region name contains a comma.
 n SOUR @<XREF:SOUR>@                       {1:1}  g7:SOUR
   +1 RESN <List:Enum>                      {0:1}  g7:RESN
   +1 PAGE <Text>                           {0:1}  g7:PAGE
+     +2 SOUR                               {0:M}  g7:SOUR
```

Comment:
This appears to be incomplete (no payload or substructures)

Also, I'm not seeing why it's under PAGE instead of SOUR

Reply:
Maybe meant
```gedcom
       +2 <<SOURCE_CITATION>>               {0:M}
```